CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ---
 
-## [Unreleased]
+## [1.0.2] - 2024-07-08
```

Comment:
**comment**: Would it make sense to update to v1.1.0 since we've added functionality in addition to fixing bugs?

Reply:
Yeah, you're right - I was just thinking of my most recent changes, but the last release was actually several months ago.

Reply:
Bumped with b7c59756242fc24d7dde5774281de7f0ea0517d0.